component-library/components/generic/component-heading/component-heading.scss

```diff
@@ -0,0 +1,73 @@
+.c-component-heading {
+  color: var(--brand-grey-000, #f9f9fb);
+  background: var(--brand-grey-900, #1b1b1d);
```

Comment:
Can we remove these 2 fields? I felt color and background color should be inherit from parent component.